myuw/templates/handlebars/card/instructor_schedule/course_sche_panel.html

```diff
@@ -55,7 +55,7 @@ <h5 class="scheduleHeading">
 </table>
 
 <h5 class="scheduleHeading">
-FINAL EXAM
+  FINAL EXAM {{#if final_exam.no_exam_or_nontraditional}}{{else}}{{#if final_exam.is_confirmed}}{{else}}(<a href="https://sdb.admin.uw.edu/sisMyUWClass/uwnetid/{{netid}}/finalexam.asp?{{capitalizeString quarter}}+{{year}}&sln={{sln}}" target="_blank">Confirm</a>){{/if}}{{/if}}
```

Comment:
since there wasn't a spec, i was just leaving it open, but unless for now is fine